gms/gossiper.cc

```diff
@@ -2275,14 +2289,19 @@ bool gossiper::is_alive(inet_address ep) const {
     if (ep == get_broadcast_address()) {
         return true;
     }
+    bool is_alive = _live_endpoints.contains(ep);
     auto* eps = get_endpoint_state_for_endpoint_ptr(ep);
     // we could assert not-null, but having isAlive fail screws a node over so badly that
     // it's worth being defensive here so minor bugs don't cause disproportionate
     // badness.  (See CASSANDRA-1463 for an example).
     if (eps) {
-        return eps->is_alive();
+        auto is_alive_state = this->is_alive_state(*eps);
```

Comment:
We should probably call `is_alive_state` only when DEBUG logging is enabled

Reply:
Will do

Reply:
Actually, a later patch deletes the hunk, so I'll just get rid of it in this patch.